code/modules/mob/living/simple_animal/slaughter/slaughter.dm

```diff
@@ -0,0 +1,433 @@
+///The following mini-antag was coded by 	KorPhaeron   for TG Station
```

Comment:
I mean, this isn't bad per say, but credits are typically un-necessary and not used over most of the code.